web/src/main/resources/org/openmrs/web/filter/initialization/adminusersetup.vm

```diff
@@ -21,7 +21,16 @@
                     <td>$l10n.get("install.advanced.admin.password.retype"):</td>
      			<td><input type="password" name="new_admin_password_confirm" value="" autocomplete="off" onKeyPress="return submitenter(this,event);"/></td>
      		</tr>
-                <tr>
+     		<tr>
+				<td colspan="2" class="question">
+						<strong>$l10n.get("install.advanced.admin.password.constraints") :</strong><br/>
+						$l10n.get("install.advanced.admin.password.constraints.min.length")<br/>
+						$l10n.get("install.advanced.admin.password.constraints.cases")<br/>
+						$l10n.get("install.advanced.admin.password.constraints.numbers")<br/>
+						$l10n.get("install.advanced.admin.password.constraints.username")<br/>
+				</td>
+     		</tr>
+				<tr>
```

Comment:
Can you do proper formatting of different tags so that they can be easily viewed? On a glimpse they appear to be jumbled.

Reply:
Done. Check it now, please.  😇
The tables itself not properly formatted. I didn't format the lines because it's hard to review that formatting fixed.